src/PolyhedralGeometry/PolyhedralComplex/constructors.jl

```diff
@@ -170,6 +170,49 @@ end
 
 polyhedral_complex(P::Polyhedron{T}) where {T<:scalar_types} = polyhedral_complex([P])
 
+@doc raw"""
+    polyhedral_complex(F::PolyhedralFan)
+
+Turn a polyhedral fan into a polyhedral complex.
+"""
+function polyhedral_complex(F::PolyhedralFan{T}) where {T<:scalar_types}
+  pmo_in = pm_object(F)
+  pmo_out = Polymake.fan.PolyhedralComplex{_scalar_type_to_polymake(T)}()
+  for prop in ["RAYS", "INPUT_RAYS", "FACET_NORMALS"]
```

Comment:
Looking at this I wondered "hmm, how many allocations will this make? Can it reuse the strings? What about the vector?"

Turns out if you change this from a vector to a tuple, you can avoid all allocations in that line. I.e.:
```suggestion
  for prop in ("RAYS", "INPUT_RAYS", "FACET_NORMALS")
```

To demonstrate the difference:
```julia
julia> function f(s)
         for prop in ["MAXIMAL_CONES", "INPUT_CONES", "MAXIMAL_CONES_FACETS"]
           s == prop && return true
         end
         return false
       end
f (generic function with 1 method)

julia> function g(s)
         for prop in ("MAXIMAL_CONES", "INPUT_CONES", "MAXIMAL_CONES_FACETS")
           s == prop && return true
         end
         return false
       end
g (generic function with 1 method)

julia> @btime f("MAXIMAL_CONES")
  25.435 ns (1 allocation: 80 bytes)
true

julia> @btime g("MAXIMAL_CONES")
  1.500 ns (0 allocations: 0 bytes)
true
```
Mind you, either is plenty fast and I assume it won't matter for `polyhedral_complex` which hopefully nobody will call in a tight loop ;-) so this is *not* a change request -- I just wondered about the behaviour of this code myself and thought I'd share my findings for future reference :-)

Reply:
Nice, thank you.

Reply:
I learned something new in julia today.  So is it generally recommended to use tuples over vectors when (easily) possible?